jenkins-slaves/jenkins-slave-npm/Dockerfile.rhel7

```diff
@@ -15,10 +15,9 @@ ENV NODEJS_VERSION=10 \
     PATH=$HOME/node_modules/.bin/:$HOME/.npm-global/bin/:$PATH \
     CHROME_BIN=/bin/google-chrome
 
-COPY contrib/bin/scl_enable /usr/local/bin/scl_enable
 ADD src/google-chrome.repo /etc/yum.repos.d/
 
-RUN INSTALL_PKGS="rh-nodejs${NODEJS_VERSION} rh-nodejs${NODEJS_VERSION}-npm rh-nodejs${NODEJS_VERSION}-nodejs-nodemon nss_wrapper redhat-lsb libXScrnSaver xdg-utils google-chrome-stable" && \
+RUN INSTALL_PKGS="npm nss_wrapper redhat-lsb libXScrnSaver xdg-utils google-chrome-stable" && \
```

Comment:
Does this assume that the OpenShift hosts has some special repo activated?

Reply:
If so, maybe we should indicate in README.md what repo(s) needs be enabled on OpenShift hosts to be able to build this image, please?

Reply:
This has been resolved